DopplerFiles/DopplerFiles.csproj

```diff
@@ -8,4 +8,8 @@
     <PackageReference Include="Swashbuckle.AspNetCore" Version="5.6.3" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\StorageProviders\StorageProviders.csproj" />
```

Comment:
I would not add a new project for this, but I have not a strong opinion.

Reply:
This project will be used for implementing AWS S3 provider